packages/rlc-common/src/interfaces.ts

```diff
@@ -135,7 +136,8 @@ export interface PagingDetails {
 }
 
 export type Methods = {
-  [key: string]: [OperationMethod];
+  // could be more than one method if overloading
+  [key: string]: [OperationMethod, ...OperationMethod[]];
```

Comment:
Is it trying to suggest there's at least one OperationMethod ? is it necessary?

Reply:
yes, it would be easier to touch the first element without checking.